packages/@webex/plugin-meetings/src/multistream/remoteMediaGroup.ts

```diff
@@ -36,13 +37,20 @@ export class RemoteMediaGroup {
     this.mediaRequestManager = mediaRequestManager;
     this.priority = priority;
     this.options = options;
+    this.mediaRequestIds = [];
 
     this.unpinnedRemoteMedia = receiveSlots.map(
       (slot) =>
         new RemoteMedia(slot, this.mediaRequestManager, {
           resolution: this.options.resolution,
         })
     );
+    this.namedMedia = this.unpinnedRemoteMedia.filter(
+      (item) => item !== null && !!item.namedMediaGroup && !!item.namedMediaGroup.value
+    );
+    this.unpinnedRemoteMedia = this.unpinnedRemoteMedia.filter(
+      (item) => item !== null && (!item.namedMediaGroup || !item.namedMediaGroup.value)
```

Comment:
I don't think `this.unpinnedRemoteMedia` can ever have any null or undefined values, can it? if it can't then this code can be simplified a bit